contact.html

```diff
@@ -54,10 +54,9 @@ <h2 class="location-header">India Development Center</h2>
           <div class="address">
             <i class="icon-location icon"></i>
             <p>
-               <strong> Infratab Bangalore Pvt Ltd </strong><br>
-               	Cowrks, Building 6A,<br> RMZ Eco World,
-                Outer Ring Rd,<br> Devarabisanahalli,
-                Bellandur, <br> Bengaluru, Karnataka 560103<br>
+              <strong> Infratab Bangalore Pvt Ltd </strong><br>
+               91 springboard, 3nd Floor,<br> Padmavathi Complex, 80 Feet Rd,
```

Comment:
@priyanka0901 should we use 2nd floor here?

Reply:
@niren777 Because office address is registered with 2nd floor.